src/openforms/js/lang/formio/nl.json

```diff
@@ -74,7 +74,8 @@
   "Edit JSON": "Wijzig JSON",
   "Advanced Logic Configured": "Geavanceerde logica geconfigureerd",
   "actions": "acties",
-  "autocomplete": "automatisch aanvullen",
```

Comment:
I'm afraid this may have broken some frontend translations due to the changed case. Can you add `Autocomplete` as a new item instead of updating this line?